LDAP/ldap2s3/ldap2s3.py

```diff
@@ -179,52 +261,39 @@ def group(self, entry):
     users = {}
     sgen = mozldap.conn.extend.standard.paged_search(search_base=config.ldap.search_base.users,
                                                      search_filter=config.ldap.filter.users,
-                                                     attributes = ['mail', 'sshPublicKey', 'pgpFingerprint'],
+                                                     attributes = ['mail', 'sshPublicKey', 'pgpFingerprint', 'sn',
+                                                                   'givenName', 'workdayTimezone', 'telephoneNumber',
+                                                                   'createTimestamp', 'modifyTimestamp'],
                                                      search_scope=SUBTREE, paged_size=10, generator=True)
+    # Create the list of all users
     for entry in sgen:
-        u = mozldap.user(entry)
```

Comment:
removed all the unused stuff and more, now that it no longer needs to create multiple userlists to match things with ldap groups